fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -296,6 +297,7 @@ public Response deleteObject() {
      * @return 204
      * @throws InvalidChecksumException if invalid checksum exception occurred
      * @throws MalformedRdfException if malformed rdf exception occurred
+     * @throws InsufficientStorageException if operation fails due to insufficient storage on the device.
```

Comment:
If this becomes a runtime exception (see below), there will be no need to declare it here or in the method signature.